pkg/minikube/download/image.go

```diff
@@ -139,38 +138,54 @@ func ImageToCache(img string) error {
 		return errors.Wrap(err, "getting remote image")
 	}
 	klog.V(3).Infof("Writing image %v", tag)
+	errchan := make(chan error)
+	// buffered channel
+	c := make(chan v1.Update, 200)
+	var p *pb.ProgressBar
 	if out.JSON {
-		if err := tarball.WriteToFile(f, tag, i); err != nil {
-			return errors.Wrap(err, "writing tarball image")
+		register.PrintDownloadProgress(img, "0")
+	} else {
+		// f we need to print progress bat to stdout
```

Comment:
```suggestion
		// if we need to print progress bar to stdout
```
Not sure if `f` is supposed to be `if` or not?